index.html

```diff
@@ -184,9 +183,42 @@ <h6> I can help you build a product, feature or website Look through some of my
                                 <ul>
                                 </div>
                               </div>
-                      </div>
-                          
-                  
+                            </div> 
+                            <section>
+                              <p class="about-p">
+                                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                              </p>
+                              <form action="https://formspree.io/f/xwkjrzvw"
+                              method="POST">
+                                <ul class="inputs">
+                                   <li>
+                                   <label for="name"></label>
+                                    <input type="text" id="name" name="user_name" placeholder="Full name"/>
```

Comment:
- [ ] Great job so far. Please ensure that All input fields have the required attribute to activate the validation on submit
- [ ] please, the Name input should have  `maxlength="30"` attribute. This will limit very long names
- [ ] kindly ensure that the Message input has `maxlength="500"` attribute.

Reply:
> * [ ]  Great job so far. Please ensure that All input fields have the required attribute to activate the validation on submit
> * [ ]  please, the Name input should have  `maxlength="30"` attribute. This will limit very long names
> * [ ]  kindly ensure that the Message input has `maxlength="500"` attribute.

- [ ] As suggested by the previous reviewer, I would implore you to add required attribute to all the inputs and textarea tags so that there would be no submission without input value. Thanks to html5 that provides this for us.
- [ ] Following the previous reviewer, I would also implore you to add maxlenght to the text input and the textarea input so that the number of character input can be controlled by you and this is part of the requirement.